docs/advanced-topics/dockstore-cli/dockstore-cli-faq.rst

```diff
@@ -17,6 +19,8 @@ FTP, S3, and GCS. As of Release 1.12, the Dockstore CLI has support for running
 client <https://docs.icgc.org/download/guide/#score-client-usage>`__. Please see `file provisioning plugins <https://github.com/dockstore/dockstore-cli/tree/master/dockstore-file-plugin-parent>`__
 for more information on these two file transfer sources.
 
+:ref:`(back to top) <topCLIFAQ>`
```

Comment:
It seems like this PR also has commits for #247 included in it. It would be good to merge that one so we can see what's really different.